translations/catroweb.en.yml

```diff
@@ -66,6 +67,11 @@ notificationsClearError: "Unable to mark notifications as read!"
 notificationsDeleteError: "Unable to delete notifications!"
 pleaseWait: "Please wait..."
 projects: "Projects"
+oauthPopUpTitle: "Oauth Sign In Information"
```

Comment:
Please use correct english grammar. (uppercase/lowercase)

Oauth sign in information